src/CLoop.cpp

```diff
@@ -92,15 +115,18 @@ void CLoop::Loop(float lumFactor, int z_sample, std::string key, const CLoopConf
             eventWeight = weight*lumFactor*mjj_w;
     
         }
-
         // fill histograms
-        //cout << eventWeight;
+        //std::cout << eventWeight<< "   ";
         if (saveHistograms) Fill(eventWeight, z_sample, key);
         if (saveEvents) FillTree(eventWeight, z_sample, key);
         // end filling
 
     }
     // end style and writing
+    if (unordered_jet > 0) std::cout<<"Warning: Sample contains unordered jets in "<<unordered_jet<<" events"<<std::endl;
+    if (unordered_tau > 0) std::cout<<"Warning: Sample contains unordered jets in "<<unordered_tau<<" events"<<std::endl;
+
```

Comment:
Maybe remove this spaces... or even the entire check now that we know that the objects are ordered.